### DISCUSSION TITLE
Enable rte onload of the element

### DISCUSSION THREAD

**Discussion by yashvi2026** [GrapesJS/grapesjs#5129] [Q&A]
can i know how to enable rte for a element defaultly without invoking ondoubleclick

**Comment by artf** [comment]
`editor.getSelected().trigger('active')`

**Reply by yashvi2026** [reply]
i want it to stay active even when clicked outside the element

**Reply by artf** [reply]
Please check the source of the [ComponentTextView](https://github.com/GrapesJS/grapesjs/blob/dev/src/dom_components/view/ComponentTextView.ts) and extend it as you wish
